src/lib/badge/badge.md

```diff
@@ -51,6 +51,22 @@ background color to `primary`, `accent`, or `warn`.
 </mat-icon>
 ```
 
+### Badge max value
+The max value badge can display can be set via `matBadgeMax` tag. Max property will be applied only
+when the badge's content is numeric. If `matBadgeMax` tag is applied and contend exceed the max value
+badge will show max value and plus symbol
+```html
+<mat-icon matBadge="22" matBadgeMax=99>
```

Comment:
Missing quotes around the max attribute.

Reply:
Thank you for this catch. I have added the quotes